src/main/kotlin/io/github/moulberry/notenoughupdates/miscfeatures/CountdownCalculator.kt

```diff
@@ -35,6 +35,12 @@ import java.time.format.DateTimeFormatter
    Intended to detect countdowns within item tooltips, converting the time remaining into
    units of seconds, and then adding that result to the user's current system time
    in Unix epoch form converted into a human-readable timestamp, timezones included.
+
+   Since its original implementation, a change as been made to add the units of time directly
+   without converting them into units of seconds first. This is an (untested) attempt to fix
+   the weird bug hannibal002 has been having where countdowns slowly desync from the
+   system time. The original logic has been preserved in a comment block in case this (untested)
```

Comment:
can be replaced by git history

Reply:
removed this paragraph, also revised previous paragraph to reflect